content/blog/release-2.3.0.md

```diff
@@ -0,0 +1,111 @@
+to QtCreator - it is now possible to open source files from *.qbs files by simply pressing F2.
+We have also added support for autocompletion of the module properties:
+
+![autocompletion](../../img/blog/release-2.3.0.image.1.png)
+
+### General
+We have added a possibility to export products to CMake via the new Exporter.cmake module. This
+allows to use libraries built with QBS directly with CMake projects without the need to use an
+intermidiate representation in form of the Exporter.pkgconfig.
+
+We deprecated the "fallback" module
+[provider](https://doc.qt.io/qbs/module-providers.html#how-qbs-uses-module-providers) in favor of
+the [qbspkgconfig](https://doc.qt.io/qbs/qml-qbsmoduleproviders-qbspkgconfig.html) provider.
+In Qbs, module providers generate missing modules during the project configuration ("resolve")
+stage. Historically, we had only Qt and "fallback" module providers and there was no way to control
+which providers are run until we introduced "named" module providers in
+[Qbs 1.21](https://qbs.io/blog/release-1.21/). At the same time, we've added the
+`qbspkgconfig` provider that reads .pc files directly, avoiding expensive invocations of the
+`pkg-config` tool. Since then, we had two almost identical providers that work on top of .pc files.
+However, the problem with the "fallback" provider was that it works differently from other
+providers as well as suffer performance issues and has the same bugs as the `pkg-config` tool.
+Thus, we decided to deprecate the "fallback" provider in Qbs 2.3.0 and remove it in Qbs 2.4.0.
+For details how to use the `qbspkgconfig`, see the
+[provider page](https://qbs.io//docs/qml-qbsmoduleproviders-qbspkgconfig/) in documentaion.
+Also, please report any bugs in the `qbspkgconfig` to the
+[bug tracker](https://bugreports.qt.io/browse/QBS/).
+
+We have added a new [Tutorial](https://qbs.io//docs/tutorial/) section in the documentaion to make
+it easier to grasp core concepts and make user familiar with best practices. Also, we added a
+new [example](https://github.com/qbs/qbs/blob/master/examples/exporters/qbs/imports/MyLibrary.qbs)
+on how to use Exporter modules.
+
+If a project needs to be re-resolved, we now print the reason. Also, we rewrote the wildards
+handling to track changes more accurate.
+
+### C/C++ Support
+We fixed an issue with complex project structure that lead to a non-linear algorithm during
+traversing of the dependencies. Now, private dependencies (dependencies of static libraries) of
+products are not traversed more than once anymore
+([QBS-1714](https://bugreports.qt.io/browse/QBS-1714)).
+
+### Language
+* Module properties are now accessible for groups in modules
+  ([QBS-1770](https://bugreports.qt.io/browse/QBS-1770)).
+* The qbspkgconfig.mergeDependencies property was removed. This feature only existed because of
+  performance limitations which were fixed in [Qbs 2.1.0](https://qbs.io/blog/release-2.1.0/) and
+  thus was removed.
```

Comment:
I'd get rid of this last part; you already mentioned the removal in the previous sentence.